src/libraries/System.Collections/tests/Generic/SortedDictionary/SortedDictionary.Generic.Tests.cs

```diff
@@ -62,6 +62,18 @@ public void SortedDictionary_Generic_Constructor_IDictionary_IComparer(int count
             Assert.Equal(comparer, copied.Comparer);
         }
 
+        [Theory]
+        [MemberData(nameof(ValidCollectionSizes))]
```

Comment:
btw, the existing test above seems to be identical to SortedDictionary_Generic_Constructor_IComparer(). You could remove the latter if you like.